marklogic-data-hub-central/ui/src/components/entities/mapping/source-entity-map/source-to-entity-map.tsx

```diff
@@ -619,7 +617,7 @@ const SourceToEntityMap = (props) => {
                                         expandIcon={(props) => customExpandIcon(props)}
                                         className={styles.sourceTable}
                                         rowClassName={() => styles.sourceTableRows}
-                                        scroll={{y: '60vh', x: 'max-content' }}
+                                        scroll={{y: '60vh'}}
```

Comment:
As discussed, let's not remove the x axis scroll as it will be helpful in cases where we have longer column names in nested table structure with more than 2 or 3 levels.